### PR TITLE
Doc(eos_designs): Addition of L2LS documentation

### PR DESCRIPTION
**Pull request by JulioPDX**
## Change Summary

Additions of L2LS design type for eos_designs

## Related Issue(s)

Fixes aristanetworks/avd-internal#51

## How to test

Documentation change only.

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by carlbuchmann — APPROVED** [review]
LGTM!

File: ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md [hunk 1]
```diff
@@ -38,32 +38,60 @@ pod_name: < POD_Name >
...
 | l2leaf             | ✘               | port-channel | none              | ✅                 | ✘                   | ✘   | ✅           | ✅                 |
 | overlay_controller | ✅              | p2p          | server            | ✘                  | ✘                   | ✘   | ✘            | ✘                  |
 
+### L2LS
+
+| Node Type Key      | Underlay Router | Uplink Type | Default EVPN Role  | L2 Network Services | L3 Network Services | VTEP | MLAG Support | Connected Endpoints |
+| ------------------ | --------------- | ------------ | ----------------- | ------------------- | ------------------- | ---- | ------------ | ------------------- |
+| l3spine              | ✅              | p2p          | none            | ✅                  | ✅                   | ✘   | ✅            | ✅                  |
+| spine             | ✘              | port-channel | none            | ✅                 | ✘                  | ✘  | ✅           | ✅                 |
+| l2leaf             | ✘               | port-channel | none              | ✅                 | ✘                   | ✘   | ✅           | ✅                 |
```
Comment (mthiel117) [inline ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md]: line 93: This should be `leaf` not `l2leaf`.

Comment (mthiel117) [reply ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md]: Look at line 79 of this file: https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-node-type-keys.yml

Comment (mthiel117) [reply ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md]: Otherwise LGTM.

**Review by mthiel117 — APPROVED** [review]
LGTM...

File: ansible_collections/arista/avd/roles/eos_designs/README.md [hunk 2]
```diff
@@ -90,6 +91,12 @@ Across all designs, the following functionality is provided:
   <img src="../../media/dissag-topology.gif" />
 </div>
 
+### Layer 2 Leaf Spine
+
+| Underlay | Overlay | Topology |
+| -------- | ------- | -------- |
+| L2 | N/A | Arbitrary or leaf-spine |
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/README.md]:
```suggestion
| Spine Uplinks | Leaf Uplinks | Topology |
| L2 | L2 | Arbitrary or leaf-spine |
| L3 + Static routes | L2 | Arbitrary or leaf-spine |
| L3 + eBGP | L2 | Arbitrary or leaf-spine |
| L3 + OSPF | L2 | Arbitrary or leaf-spine |
| L3 + ISIS | L2 | Arbitrary or leaf-spine |
```

**Review by carlbuchmann — APPROVED** [review]

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**